.devcontainer/devcontainer.json

```
@@ -29,9 +22,7 @@
 			"version": "8.0",
 			"additionalVersions": "6.0"
 		},
-		"ghcr.io/azure/azure-dev/azd:latest": {
-			"version": "1.9.3"
-		},
+		"ghcr.io/azure/azure-dev/azd:latest": {},
```

```suggestion
		"ghcr.io/azure/azure-dev/azd:latest": {
			"version": "1.9.4"
		},
```

we want to pin the version (we've been bit by auto-upgrades in the past)